x-pack/plugins/security_solution/public/common/components/link_to/redirect_to_kubernetes.tsx

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { KUBERNETES_PATH } from '../../../../common/constants';
+import { appendSearch } from './helpers';
+
+export const getKubernetesUrl = (search?: string) => `${KUBERNETES_PATH}${appendSearch(search)}`;
```

Comment:
@semd or @machadoum do we need to do anything else here for tracking/toggling the search value. If I remember correctly, that might be implemented for the group navigation?

Reply:
By default, the URL state will be enabled. 
The disable the URL state for the old menu, one must update this function:
https://github.com/elastic/kibana/blob/3e8e89069f9c244cf7b077e3f74d04a27fb68073/x-pack/plugins/security_solution/public/common/components/link_to/index.ts#L90

For the new menu, it is a property like here https://github.com/elastic/kibana/blob/3e8e89069f9c244cf7b077e3f74d04a27fb68073/x-pack/plugins/security_solution/public/landing_pages/links.ts#L34